default.nix

```
@@ -1,9 +1,18 @@
-{ system ? null, allInOne ? true, incremental ? false, static ? false, img_tag ? "", tag ? "", img_org ? "" }:
+{ system ? null, allInOne ? true, incremental ? false, static ? false, img_tag ? "", tag ? "", img_org ? "", product_prefix ? "" }:
```

maybe null? just incase we ever want to build without a prefix and below we compare against null rather than ""

```suggestion
{ system ? null, allInOne ? true, incremental ? false, static ? false, img_tag ? "", tag ? "", img_org ? "", product_prefix ? null }:
```

---

Sure, I think that makes sense.